financial-services-accelerator/accelerators/fs-apim/repository/resources/apis/Accounts/accounts-dynamic-endpoint-insequence.xml

```diff
@@ -23,15 +23,15 @@
     <property name="Authorization" scope="transport" action="remove"/>
     <!-- Replace the  <AUTH_HEADER_VALUE> with Base64 encoded "username:password" value.
     i.e Base64(username:password) -->
-    <property name="Authorization" value="<AUTH_HEADER_VALUE>" scope="transport"/>
+    <property name="Authorization" value="Basic YWRtaW5Ad3NvMi5jb206d3NvMjEyMw==" scope="transport"/>
```

Comment:
We can use the BasicAuthMediator to retrieve this within the sequence.

Reply:
Reverted to original placeholder